test/stats.node.js

```diff
@@ -82,19 +82,47 @@ describe('Stats', () => {
     })
   })
 
-  it('waits a bit', (done) => setTimeout(done, 1900))
+  it('waits a bit', (done) => setTimeout(done, 1000))
```

Comment:
A test case that just waits? Should fix the tests to not depend on the ordering...

Reply:
@VictorBjelkholm good call! Fixed by moving the preparation code to a before hook.

Reply:
@pgte better to get rid of the after/before hooks we have and have function calls in the test cases that returns what you need. If all tests where shaped like this, we can run tests in parallel and just run one isolated test case if wanted.

Reply:
In my case I need something to run once, not for every test.
In this case, I want to setup some switches, exchange some protocol-specific data, wait for the stats to compute, and *then* I can infer all the statistics are there.
@VictorBjelkholm Any recommendations?

Reply:
@VictorBjelkholm changed to isolated tests for stats, would love your feedback.